updating/updating-cluster-cli.adoc

```diff
@@ -34,6 +35,8 @@ include::modules/update-service-overview.adoc[leveloffset=+1]
 
 include::modules/understanding-upgrade-channels.adoc[leveloffset=+1]
 
+include::modules/machine-health-checks-pausing.adoc[leveloffset=+1]
```

Comment:
Please review the other installation assemblies and confirm with engineering that this is the only one where this scenario applies. You might need to add this module to more of the upgrading assemblies.

Reply:
@kalexand-rh Thank you for reviewing the updated content. I will check this one with engineering. However, I'd like to confirm if you meant _upgrade assemblies_ instead of installation assemblies in this comment (...review the other installation assemblies...).

Reply:
Hi @kalexand-rh I reviewed the reviewed the upgrade assemblies with @slintes and concluded that the current one is the only one where we need to add this module.

Reply:
> the current one is the only one where we need to add this module.

@kalexand-rh and @slintes - Reassessing the upgrade assemblies. Kindly hold the review. Sorry for the inconvenience.

Reply:
@kalexand-rh Completed reassessing the assemblies and made relevant changes. You may now review the updates. Thanks.